src/bosquet/openai.clj

```diff
@@ -12,14 +11,6 @@
 #_:clj-kondo/ignore
 (def cgpt "gpt-3.5-turbo")
 
-(defn- get-api-key
```

Comment:
This is not needed any more in bosquet.
A user could code something like this himself, if he wants.